NitroxModel/Helper/NitroxUser.cs

```diff
@@ -93,14 +95,21 @@ public static string GamePath
                     return gamePath;
                 }
 
-                List<string> errors = new();
-                string path = GameInstallationFinder.Instance.FindGame(errors);
-                if (!string.IsNullOrWhiteSpace(path) && Directory.Exists(path))
+                string path = null;
+                List<GameFinderResult> finderResults = GameInstallationFinder.Instance.FindGame(GameInfo.Subnautica).TakeUntilInclusive(r => r is { IsOk: false }).ToList();
```

Comment:
Shouldn't this be `isOk: true`?

Reply:
No, but I agree it looks weird. It's purpose is to "take all the bad results and stop + 1". This way we can log the bad results instead of having to run the `FindGame` twice.